### ISSUE TITLE
Can't load shader error when importing an GLTF/GLB files with Unity 2021.3.3f1

### ISSUE THREAD

**Issue by insthync** [atteneder/glTFast#408]
**Describe the bug**
Can't load shader error when importing an GLTF/GLB files with Unity 2021.3.3f1, I can fix by myself by load the source from repo and change codes in `ShaderGraphMaterialGenerator.cs`

**To Reproduce**
- Import GLTF/GLB files in Unity editor

**Expected behavior**
Be able to import GLTF/GLB files with loaded shader

**Screenshots**
![Screenshot_1097](https://user-images.githubusercontent.com/2697550/171324936-c3ca9e9e-1633-4d4c-ab8d-6fb3dff4d714.png)

**Desktop (please complete the following information):**
 - glTFast version
 - 2021.3.3f1
 - Universal Render Pipeline 12.1.16
 - Editor
 
**Additional context**
It might have something changed in Unity Editor 2021.3.3f1

**Comment by atteneder** [comment]
@insthync I'm aware that you install via Package Manager, but there are options to that

- from a local folder (where you cloned the repository into)
- from a GIT URL
- via OpenUPM installer (as recommended in the README.md file)

**Comment by insthync** [comment]
Installed from GIT URL

here is downloaded result (the folder name is `glTFast` not `com.atteneder.glfast`)

![Screenshot_1098](https://user-images.githubusercontent.com/2697550/171431477-6bce3fd6-4235-44eb-80fe-05fdc663bc20.png)

**Comment by atteneder** [comment]
TL;DR: Please install via OpenUPM or (if you want to develop glTFast) clone it manually and add the package from disk.

The folder shown in the project view is not the actual folder the package resides in. The actual folder (in my test project) is (relative to the project root):

`Library/PackageCache/com.atteneder.gltfast@4b9622d81e`

Notice that when you install from GIT, Unity adds the used GIT commit hash at the end. This is why it is not working when installing this way. I'll have to figure out how to get the actual package dir in any case.

Thanks.

**Comment by atteneder** [comment]
I have to rewind my last statement.

I've actually tested design-time import with a GIT installed glTFast and it just worked.

Though the actual path is suffixed with the GIT hash, the `PackageInfo.assetPath` is just `Packages/com.atteneder.gltfast`.

I have no idea nor time to figure where/how you configured your project wrong. I'll close now, but feel free to add more useful info and re-open, if you find the issue.

Please just follow the recommendations for installing otherwise.

Thanks
